### DISCUSSION TITLE
lange 不知道的情况下怎么使用识别图片里面的文字。

### DISCUSSION THREAD

**Discussion by guochunjiang** [PaddlePaddle/PaddleOCR#12633] [Q&A]
我现在有很多图片，需要在后端运行。并且不知道图片里面有那种语言的。
试了一下paddleclas 效果并不好。

**Comment by guochunjiang** [comment]
调用识别函数的时候，需要传lange，图片文字的语言，在我后端处理的时候，不知道图片里面包含那种语言。没办法解决

**Comment by shiboyang** [comment]
对啊 这种问题怎么处理

**Comment by BuaaAlban** [comment]
请问paddleocr有计划支持加一个语种检测模块再喂到后续的ocr模型嘛？
